doc/nrf/release_notes.rst

```diff
@@ -20,7 +20,7 @@ This page is included only in the latest documentation, because it might contain
    :maxdepth: 1
    :caption: Subpages:
 
-   releases/release-notes-changelog
+   .. releases/release-notes-2.1.0
```

Comment:
Should be taken out of the comment mode after adding the release notes-2.1.0.

Reply:
Added a PH for 2.1.0 release notes, currently empty.